scripts/add_export.py

```diff
@@ -180,13 +180,14 @@ def create_export_profile():
             )
 
     # Resample to temporal resolution defined in wildcard "sopts" with pandas resample
-    sopts = snakemake.wildcards.sopts.split("-")
-    export_profile = export_profile.resample(sopts[0].casefold()).mean()
+    sel_export = export_profile[n.snapshots]
+    pu_profile_export = sel_export / (1e-6 + sel_export.sum())
+    export_profile = pu_profile_export * export_profile.sum() * len(n.snapshots) / 8760
```

Comment:
Is there a specific reason why the pandas resampling function is dropped here? How is it done in other scripts?